ega-data-api-res/src/main/resources/application.yml

```diff
@@ -1,7 +1,14 @@
-spring.profiles: no-oss
+spring.profiles: no-oss,LocalEGA,log-transfer,repo-logger,db-repo-logger
```

Comment:
The `spring.profiles.include` why not used here as in above `dataedge/application.yml`?

Reply:
because it made the integration tests fail for some reason :) or when pushed with `spring.profiles.include` the integration tests failed, or maybe I did something wrong.

Reply:
I will do it like that and see what we get.

Reply:
Done

Reply:
@anandmohan777 seems to fail:
```
[ERROR] testGetArchiveFile(eu.elixir.ega.ebi.it.res.ResControllerIntgTest)  Time elapsed: 0.087 s  <<< ERROR!
java.net.ConnectException: Connection refused (Connection refused)
	at eu.elixir.ega.ebi.it.res.ResControllerIntgTest.testGetArchiveFile(ResControllerIntgTest.java:30)
[ERROR] testGetFile(eu.elixir.ega.ebi.it.res.ResControllerIntgTest)  Time elapsed: 0.035 s  <<< ERROR!
java.net.ConnectException: Connection refused (Connection refused)
	at eu.elixir.ega.ebi.it.res.ResControllerIntgTest.testGetFile(ResControllerIntgTest.java:41)
```